docs/new-contributors.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
```

Comment:
Nit-pick - only need a single whitespace after the list bullet - repeated elsewhere in the file